sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Added experimental implementation of [`sycl_ext_intel_grf_size`](doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc) [370aa2a01711]
+- Added experimental implementation of [`sycl_ext_oneapi_bindless_images`](doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) [58a8f2001d8b]
+- Experimental [sycl_ext_oneapi_device_global](doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc) extension is supported now. [d812d1e13ab5]
+- Enabled media API that works with image accessors for ESIMD. Those accessors do not depend on stateful/stateless mode. [b95b2d88e446]
+- Added complex literals support to `sycl::ext::oneapi::experimental::complex`. [83555cd511cc]
+- Allowed implicit conversion from `std::experimental::simd_mask` to `ESIMD::simd_mask`. [de5b479a2de9]
+- Added `host_task` image accessor support. [b6faefb08e23]
+- Started to support 64-bit offsets with accessors in stateless mode for ESIMD APIs. [f28ae00bf6a3][d04ebb03c1c8][62955fc20e14]
+- Added `__imf_llabs` to the imf device libraries. [3c19581f828c]
+- Added explicit conversion of `multi_ptr<T>` to `multi_ptr<const T>`. [10d2f5b613f3]
+- Added support for scalar logical operators with group algorithms. [11ac73003056][90406b2ba07c]
+- Started to print supported SG sizes in `sycl-ls --verbose`. [12515177ab03]
+- Implemented device query for 64 bit atomic support in ESIMD emulator. [c40baa6db64b]
+- Added support of 16 bit data for `lsc_atomic_update` and `lsc_slm_atomic_update` ESIMD API. [3028d82a75d2]
+- Added support for per-kernel auto GRF mode specification, and reimplemented the feature using kernel properties. Introduced a new property named `sycl::detail::register_alloc_mode` which takes in an enum `sycl::detail::register_alloc_mode_enum` with values: `automatic` and `large`. [f363bb272b5a]
+- Implemented [`sycl_ext_oneapi_root_group`](doc/extensions/proposed/sycl_ext_oneapi_root_group.asciidoc) extension. [743c35be2da7]
+- Added generic implementation for some bf16 math functions. [c7759bb8cd0b]
+- Added decorated async_work_group_copy overloads. [d1f67ca228f2]
+- Added initial implementation of the Unified Runtime plugin and routed to it with `SYCL_PREFER_UR`. [5a239c6a08e1]
+- Added support for accessors to atomic_update ESIMD API. [59a1aa0841da]
+- Enabled pretty-printing of reference objects. [e3fc16f56c0e]
+- Added Xmethods that help GDB to support reading local_accessors on GPU from SLM. [ddfc5220afaa]
+- Enabled passing local accessors to ESIMD kernel and via `invoke_simd()` API, enabled usage of `get_pointer()` and `operator[]` for accessors in ESIMD kernel. The support of accessor and local_accessor is still limited comparing to SYCL. [82ae85d12bf0]
+- Implement unpadding for 2d block load/store ESIMD API. [9be8a306d0eb]
+- Enabled improved code location for SYCL extensions. [4d73490220c8]
```

Comment:
Seems like an internal change which enables runtime developers to add XPTI tracing features to extension, but doesn't bring any immediate benefits to users. Should we omit it maybe?

@reble: FYI